README.md

```diff
@@ -64,9 +64,9 @@ The single required source, file `json.hpp` is in the `single_include/nlohmann`
 using json = nlohmann::json;
 ```
 
-to the files you want to use JSON objects. That's it. Do not forget to set the necessary switches to enable C++11 (e.g., `-std=c++11` for GCC and Clang).
+to the files you want to use JSON objects and set the necessary switches to enable C++11 (e.g., `-std=c++11` for GCC and Clang).
 
-You can further use file [`include/nlohmann/json_fwd.hpp`](https://github.com/nlohmann/json/blob/develop/include/nlohmann/json_fwd.hpp) for forward-declarations. The installation of json_fwd.hpp (as part of cmake's install step), can be achieved by setting `-DJSON_MultipleHeaders=ON`:
+You can install [`include/nlohmann/json_fwd.hpp`](https://github.com/nlohmann/json/blob/develop/include/nlohmann/json_fwd.hpp) by setting `-DJSON_MultipleHeaders=ON` as a part of cmake's install step for forward-declarations.
```

Comment:
`-DJSON_MultipleHeaders=ON` is not for installing forward-declarations - this is just a side effect. I wanted to note there *exists* a forward-declaration header. And that it is already part of the CMake project with those flags.

Reply:
Sorry about this. If this is the case, I will keep the original version.

Reply:
OK.